code/modules/projectiles/guns/mounted.dm

```diff
@@ -1,12 +1,11 @@
 /obj/item/weapon/gun/energy/gun/advtaser/mounted
 	name = "mounted taser"
 	desc = "An arm mounted dual-mode weapon that fires electrodes and disabler shots."
-	icon_state = "armcannon"
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "taser"
+	item_state = "armcannonstun4"
 	force = 5
 	selfcharge = 1
-	flags = NODROP
```

Comment:
its handled on the organ side.
so that any item can be added to the implant without worrying about forgetting to set the flags.